docs/apis-tools/administration-api/administration-api-reference.md

```diff
@@ -6,6 +6,10 @@ description: "Create and manage clusters, and interact with Camunda 8 management
 sidebar_position: 1
 ---
 
+:::note
```

Comment:
Since the Admin API is not included in the Camunda 8 REST API, I'm hesitant to include a note referring to the V1 and V2 APIs.

Reply:
Good point, @akeller, I agree 👍